src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,60 @@
 import React from 'react';
 
 import './MovieCard.scss';
+import { Movie } from '../../types/Movie';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
-);
+type Props = {
```

Comment:
```suggestion
interface MovieCardProps = {
```
lets change Props to something more descriptive, also use interface instead of type :)